clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -21067,6 +21067,10 @@ Sema::ActOnOpenMPDependClause(const OMPDependClause::DependDataTy &Data,
               ExprTy = ATy->getElementType();
             else
               ExprTy = BaseType->getPointeeType();
+            // bug 69200
+            if (ExprTy.isNull()) {
+              continue;
+            }
```

Comment:
```suggestion
            if (ExprTy.isNull())
              continue;
```
LLVM style omits braces on a single line block.